storybook/stories/API/tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { pageData } from '../../data';
+import { Line, LineChart, ResponsiveContainer, Tooltip } from '../../../../src';
+
+export default {
+  component: LineChart,
+};
+
+export const SimpleTooltip = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <LineChart {...args}>
+          <Line dataKey="uv" />
+          <Tooltip />
+        </LineChart>
+      </ResponsiveContainer>
+    );
+  },
+  args: {
+    data: pageData,
+    dataKey: 'uv',
+  },
+};
```

Comment:
```suggestion
export const SimpleTooltip = {
  render: (tooltipArgs: Record<string, any>) => {
    return (
      <ResponsiveContainer width="100%" height={400}>
        <LineChart data={pageData}>
          <Line dataKey="uv" />
          <Tooltip {...tooltipArgs} />
        </LineChart>
      </ResponsiveContainer>
    );
  },
};
```

In the API section of storybook, we want to document the API of a single component, in this case, the Tooltip. 

Thus every story should only have args that belong to the component in question.

Reply:
You could consider the Line stories as an example. There, we group the props by categories (i.e. animation, style, general, ...) and create a story for each, with non default values, which show the behaviour and are changeable per story.

Reply:
okay, let me explain the suggested changes, so that I can be certain if I understood the required changes to be made.

- the tooltip story currently having args that is not relevant to the component, so I need to change the args that are specific to the tooltip component and will displayed in the controls section of the storybook playground.

- If I understand the problem correctly, I need to use "tooltipArgs" for that case.

Reply:
Yes, makes sense.

Reply:
I was checking out the Line stories, and understood the problem in this PR, but I couldn't find any tooltipArgs on the API props, do I need to or can I import **TooltipProps** from the tooltip component

Reply:
tooltipPops is not a type just an alias to make clear that we only want to pass it to the tooltip in api stories